### DISCUSSION TITLE
Scheduler for RxRx1 ERM Baseline

### DISCUSSION THREAD

**Discussion by niki-amini-naieni** [p-lambda/wilds#119] [Q&A] [answered]
Hello,

I am wondering if the learning rate of the model used for RxRx1's ERM baseline was updated every batch or every epoch.

Here is the model description from the paper for reference.
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/61301874/163044139-27b2388d-efc4-4469-be1e-899364783db8.png">
[wilds paper](https://arxiv.org/pdf/2012.07421.pdf)

Thank you for your time and help.

**Comment by ssagawa** [comment, marked as answer]
The learning rate is updated in each batch.

**Reply by niki-amini-naieni** [reply]
Thank you!
